config/settings.yml

```diff
@@ -16,6 +16,7 @@
   :ldaphost:
   :ldapport: '389'
   :mode: database
+  :group_delimiters: ";:,"
```

Comment:
@kbrock would it simplify things to expose only a single character as a group delimiter per authenticator type, or perhaps for all authenticators?

Would that decrease the opportunity for a regex security or a typo?

In other words, something like this but please clean it up:

```ruby
groups = if httpd_group_delimiter.to_s.length == 1
  CGI.unescape(user_headers['X-REMOTE-USER-GROUPS'] || '').split(httpd_group_delimiter.to_s)
else
  CGI.unescape(user_headers['X-REMOTE-USER-GROUPS'] || '').split(/[;:,]/)
end

Reply:
Note, I thought it was weird that we support multiple delimiters based on whichever ldap system we were adding at the time.  It feels like each "type" could have the current values.  I'm not sure which uses `;` or `:` or `,`.  Either way, if we keep the existing behavior as the default if you don't specify a delimiter, it shouldn't break anything unless you opt-in and use the wrong value for your system.

Reply:
updated comment to describe which system uses which delimiter
Also modified the configuration to make it more obvious